src/pages/my_page/user_profile/ProfilePictureUploader.tsx

```diff
@@ -52,7 +56,7 @@ export const ProfilePictureUploader = ({ personId }: ProfilePictureUploaderProps
         <Box sx={{ display: 'flex', flexDirection: 'column' }}>
           <IconButton
             data-testid={dataTestId.myPage.myProfile.deleteProfilePictureButton}
-            onClick={() => mutateProfilePicture.mutate('')}
+            onClick={() => toggleConfirmDialog()}
```

Comment:
gevinsten med å lage en `toggle`-funksjon er at vi slipper å lage nye funksjoner for hver event handler, men her wrappes den bare i en ny funksjon likevel
```suggestion
            onClick={toggleConfirmDialog}
```